### PR TITLE
Polygon switch layer

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/pages/Search.jsx [hunk 1]
```diff
@@ -1441,7 +1455,7 @@ class Search extends Component {
       newState.rasterUrls = [];
       newState.searchType = "definedArea";
       newState.polygon = null;
-      newState.polygonArea = 0;
+      /* newState.polygonArea = 0; */
```
Comment (ManuelStardust) [inline src/pages/Search.jsx:1458]: This line is commented but it is no longer useful

**Review by ManuelStardust — CHANGES_REQUESTED** [review]

File: src/pages/Search.jsx (hunk 1 above)
Comment (cazapatamar) [reply src/pages/Search.jsx]: Fixed!

File: src/pages/search/SearchContext.tsx [hunk 2]
```diff
@@ -6,10 +6,8 @@ export interface SearchContextValues {
   geofenceId: string | number;
   searchType: "definedArea" | "drawPolygon";
   polygon: Polygon | null;
-  polygonFolder: string;
-  polygonArea: number;
   setPolygonArea(value: number): void;
-  switchLayer(layer: string): void;
+  switchLayer(layer: string | undefined | null): void;
```
Comment (erikasv) [inline src/pages/search/SearchContext.tsx:10]: these options are not necessary

**Review by ManuelStardust — APPROVED** [review]

**Review by erikasv — CHANGES_REQUESTED** [review]
Sorry I forgot to ask the changes in the comment I left before.

File: src/pages/search/SearchContext.tsx (hunk 2 above)
Comment (cazapatamar) [reply src/pages/search/SearchContext.tsx:10]: As `switchLayer` has these cases (`undefined` and `null`) i change here the options that function could receive.

Comment (cazapatamar) [reply src/pages/search/SearchContext.tsx]: Fixed!

**Review by erikasv — APPROVED** [review]

**Pull request merged**